src/Nethermind/Nethermind.Runner/configs/chiado.cfg

```diff
@@ -5,36 +5,23 @@
   },
   "Sync": {
     "FastSync": true,
-    "FastBlocks": true,
-    "PivotNumber": 236689,
-    "PivotHash": "0xd85c7d023943fc84a1e57e7dab19b1c87a7dbe29915376b09cf7a660ae9dfa5e",
-    "PivotTotalDifficulty": "70561291887092720722228821971658887763518365",
     "FastSyncCatchUpHeightDelta": 10000000000
   },
   "JsonRpc": {
     "Enabled": true,
     "Timeout": 20000,
     "Host": "127.0.0.1",
-    "Port": 8545,
-    "AdditionalRpcUrls": [
-      "http://localhost:8551|http;ws|net;eth;subscribe;engine;web3;client"
-    ]
-  },
-  "AuRaMerge": {
-    "Enabled": true
+    "Port": 8545
   },
   "Aura": {
```

Comment:
potentially EnablePrivateCHains for starting network, but this can be set on 1 bootnode for example.